FlowCrypt/Controllers/Inbox/InboxViewController.swift

```diff
@@ -152,6 +154,51 @@ extension InboxViewController {
                 ) { [weak self] in self?.handleSearchTap() }
             ]
         )
+        navigationItem.leftBarButtonItem = getSideMenuNavButton()
+    }
+
+    func setupThreadSelectNavigationBar() {
+        let selectedThreads = inboxInput.filter(\.isSelected)
+        navigationItem.setAccessibility(id: "\(selectedThreads.count)")
```

Comment:
let's also add `selected` to screen title, so it'll show `2 selected`, not just `2`

I noticed that Gmail app also shows just number of selected threads, but I think adding `selected` will be useful

Reply:
I'm afraid if we add   `selected` text, it will overlap with other menu buttons in small screens.

Reply:
I tried it on iPhone 8 simulator and seems to be enough space for buttons and title:

<img width="360" alt="Screenshot 2023-08-21 at 22 18 11" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/387a4591-3aeb-4a66-a5ab-4914f9a0f5af">

Reply:
Sounds good. let me add that text